README.md

```diff
@@ -35,25 +35,31 @@ routes.forEach(route => {
 })
 ```
 
-### Version Configuration
-By default, the service and layer metadata endpoints will respond with the following version attributes:
-```json
-currentVersion: 10.51,
-fullVersion: '10.5.1'
+### Setting defaults at runtime
+FeatureServer allows several defaults to be set at runtime via Express's `app.locals` method.  Specifically, you will need to set:
+
+```js
+app.locals.config = {
+  featureServer: {
+    // define default here
+  }
+}
 ```
 
-You can alter these defaults by setting properties on Express's `app.locals`:
+If you are using FeatureServer as part of a Koop instance, the equivalent of Express's `app.locals` is `koop.server.locals`.
+
+The follow properties can be set at runtime with the noted method:
 
 ```js
 app.locals.config = {
   featureServer: {
-    currentVersion: 10.81,
-    fullVersion: '10.8.1'
+    currentVersion: 11.01, // defaults to 10.51
+    fullVersion: '11.0.1', // defaults to '10.5.1'
+    serviceDescription: 'default service description',
+    description: 'default layer description'
```

Comment:
should we rename this to layerDescription?

Reply:
we could but, I'm trying to keep the property names identical to how they appear in the feature-server responses.  So the server-metadata response spec uses `serviceDescription` and the layer-metadata response spec uses `description`.  That's why they're defined like this.